### ISSUE TITLE
Upload file "internal error" (v0.5.3)

### ISSUE THREAD

**Issue by aboulinguez** [coryrwest/B2.NET#15]
Hi @coryrwest,

I've just updated my B2.NET package from 0.0.4.6 to the current 0.5.3 version and I encounter an "internal error" when I'm trying to upload a file.

My code:

```
this.options = new B2Options()
{
	AccountId = accountId,
	ApplicationKey = applicationKey,
	BucketId = bucketId,
	PersistBucket = true
};

this.client = new B2Client(B2Client.Authorize(this.options));

var uploadUrl = await client.Files.GetUploadUrl(this.options.BucketId);

//Method parameters: string fileName, byte[] data
//THE EXCEPTION OCCURED HERE
var file = await client.Files.Upload(data, fileName, uploadUrl, false, this.options.BucketId);
```

The error:

B2Exception {"internal error"} Status 500

`B2Net.Utilities.CheckForErrors(HttpResponseMessage response)\r\n   à B2Net.Http.ResponseParser.<ParseResponse>d__0`1.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   à B2Net.Files.<Upload>d__11.MoveNext()\r\n--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---\r\n   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   à System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n`

Any idea?

Thank you for your help.

Best regards,

Antoine.

**Comment by coryrwest** [comment]
I noticed that during my tested, but it was inconsistent and only happened for large files for me. I suggest downgrading back to your previous version while I work it out. I have a support case with B2 open right now to resolve this.

**Comment by aboulinguez** [comment]
Ok, I will downgrade my package version. Let me know if you have any update from B2.
Thanks for your answer and for your work.
Antoine.

**Comment by coryrwest** [comment]
This is fixed in 0.5.31
